### PR TITLE
Removal of the Application Registry, Connector Service and Connection Token Handler sources

### PR DESCRIPTION
**Pull request by mvshao**
**Description**
We are planning to remove the Application Registry, Connector Service, Connection Token Handler, Application Connectivity Cert Setup Job and Application Connector Tests,  so in this case, we won't need these components anymore. 

Changes proposed in this pull request:

- adjust the Connectivity Validator in non OS flow
- remove the Application Registry, Connector Service, Connection Token Handler Application Connectivity Cert Setup Job and Application Connector Tests sources

**Related issue(s)**
#13718

### CODE REVIEW

File: components/application-connectivity-validator/cmd/applicationconnectivityvalidator/options.go [hunk 1]
```diff
@@ -92,14 +86,14 @@ func (o *options) String() string {
 	return fmt.Sprintf("--proxyPort=%d --externalAPIPort=%d --tenant=%s --group=%s "+
 		"--eventingPathPrefixV1=%s --eventingPathPrefixV2=%s --eventingPublisherHost=%s "+
 		"--eventingPathPrefixEvents=%s --eventingDestinationPath=%s "+
-		"--appRegistryPathPrefix=%s --appRegistryHost=%s --appName=%s "+
+		"--appName=%s "+
```
Comment (akgalwas) [inline components/application-connectivity-validator/cmd/applicationconnectivityvalidator/options.go:89]: Be aware that removing this from the application's code is not enough. It should be removed from deployment.

File: components/central-application-connectivity-validator/cmd/centralapplicationconnectivityvalidator/centralapplicationconnectivityvalidator.go [hunk 2]
```diff
@@ -104,8 +104,6 @@ func main() {
 		options.eventingPathPrefixEvents,
 		options.eventingPublisherHost,
 		options.eventingDestinationPath,
-		options.appRegistryPathPrefix,
```
Comment (akgalwas) [inline components/central-application-connectivity-validator/cmd/centralapplicationconnectivityvalidator/centralapplicationconnectivityvalidator.go:107]: Be aware that removing this from the application's code is not enough. It should be removed from deployment.

**Review by akgalwas — DISMISSED** [review]

File: components/central-application-connectivity-validator/README.md [hunk 3]
```diff
@@ -3,7 +3,7 @@
 ## Overview
 
 The Central Application Connectivity Validator validates client certificate subjects.
```
Comment (majakurcius) [inline components/central-application-connectivity-validator/README.md:5]:
```suggestion
The Central Application Connectivity Validator validates client certificate subjects in the Compass mode of Kyma.
```

**Review by majakurcius — DISMISSED** [review]

**Review by majakurcius — DISMISSED** [review]

**Review by koala7659 — CHANGES_REQUESTED** [review]
Please remove also sources for component `application-connectivity-certs-setup-job `

**Review by koala7659 — DISMISSED** [review]
LGTM

**Review by koala7659 — APPROVED** [review]
LGTM

**Review by franpog859 — APPROVED** [review]

**Review by grego952 — APPROVED** [review]

**Review by klaudiagrz — APPROVED** [review]

**Pull request merged**